development-docs/KAFKA_VERSIONS.md

```diff
@@ -0,0 +1,38 @@
+# Adding / removing Kafka versions
+
+## Adding new Kafka versions
+
+There is no _definitive_ guide as every version might bring surprises.
+But these are some of the tasks you usually have to do.
+
+* If you add a new minor version, add new 3rd party libs directory to `docker-images/artifacts/kafka-thirdparty-libs`
+  * This directory is typically shared by all patch releases which are part of the same minor release. So it is named `X.Y.x` - e.g. `1.1.x`
+  * Add a `pom.xml` file with dependencies and make sure they are aligned with the dependencies inside Kafka (e.g. use similar version of Jackson libraries)
+  * (Typically, you would do this by copying the directory for the previous minor version)
+* Add the new version to `kafka-versions.yaml` file
+  * If the version you are adding is the latest, mark it as default and unmark the previous one to not be default anymore
```

Comment:
maybe more explicitly "mark it as `default: true` and unmark (as `default: false`) the previous ..." or something similar?